src/snowflake/cli/_plugins/nativeapp/run_processor.py

```diff
@@ -72,6 +72,22 @@
 }
 
 
+def print_warnings(create_or_upgrade_cursor: Optional[SnowflakeCursor]):
+    """
+    Shows warnings in the console returned by the CREATE or UPGRADE
+    APPLICATION command.
+    """
+    if not create_or_upgrade_cursor:
+        return
+
+    messages = [row[0] for row in create_or_upgrade_cursor.fetchall()]
+    warnings = [m for m in messages if m.lower().startswith("warning:")]
+    if warnings:
+        for warning in warnings:
+            cc.warning(warning)
+        cc.message("")
```

Comment:
Why is this needed?

Reply:
As mentioned in the PR description, it's to make it easy to tell there are warnings. There's so much irrelevant output it's hard to actually see this, which is very high signal, vs a lot of the rest of the noise.